src/components/AdminSettings/SignalingServer.vue

```diff
@@ -144,7 +144,10 @@ export default {
 				} else if (exception.response.data.ocs.data.error === 'JSON_INVALID') {
 					this.errorMessage = t('spreed', 'Error: Server did not respond with proper JSON')
 				} else if (exception.response.data.ocs.data.error === 'UPDATE_REQUIRED') {
-					this.errorMessage = t('spreed', 'Error: Server needs to be updated to be compatible with this version of Talk')
+					this.versionFound = exception.response.data.ocs.data.version || 'could not get version'
```

Comment:
_could not get version_ should be translatable (and ideally maybe it would be better to have another full string with the error rather than replacing it in the generic string). But as mentioned above, unless I am missing some scenario, it would happen only when the signaling server version is < 0.0.3, so... it is probably fine even if it is not translated.

Reply:
changed. Since the placeholder is in a `xxx: …; yyyy` situation that works fine